src/main/scala/kiambogo/scrava/models/SegmentEffort.scala

```diff
@@ -18,7 +18,7 @@ case class SegmentEffort(
   end_index: Int,
   average_heartrate: Option[Float],
   max_heartrate: Option[Int],
-  segment: SegmentSummary,
+  segment: Option[SegmentSummary],
```

Comment:
Is this indeed optional? From here it doesn't appear to be https://strava.github.io/api/v3/efforts/

Reply:
Yes, I got an activity without a segment and had a deserialization error.  Maybe it is because the run was imported from elsewhere or something.  It may be rare, but it is possible.

Reply:
👍